app/enterprise/2.2.x/developer-portal/administration/application-registration/enable-application-registration.md

```diff
@@ -46,56 +46,45 @@ In Kong Manager, access the Service for which you want to enable Application Reg
 
    ![Create application-registration plugin](/assets/images/docs/dev-portal/create-app-reg-plugin-form.png)
 
-6. Click **Create**.
-
-### Application Registration Configuration Parameters
-
-#### Service
-
-**Required**
-
-Select the Service that this plugin configuration will target.
-
-#### Tags
-
-An optional set of strings for grouping and filtering, separated by commas.
-
-#### Auto Approve {#aa}
-
-Default: `false`
-
-If enabled, all new Service contract requests are automatically
-approved. Otherwise, Dev Portal admins must manually approve requests.
+   **Important:** Exposing
+   the Issuer URL is essential for the
+   [Authorization Code Flow](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/3rd-party-oauth/#ac-flow)
+   workflow configured for third-party identity providers.
 
-#### Description
+   ![Issuer URL](/assets/images/docs/dev-portal/dev-portal-issuer-url.png)
 
-Default: none
-
-Unique description displayed in the information about a Service in the Developer Portal.
-
-#### Display Name
-
-**Required**
+6. Click **Create**.
 
-Unique display name used for a Service in the Developer Portal.
+### Application Registration Configuration Parameters {#app-reg-params}
 
-#### Show issuer {#show-url-issuer}
+| Form Parameter | Description                                                                       |
+|:---------------|:----------------------------------------------------------------------------------|
+| `Service` | The Service that this plugin configuration will target. Required. |
+| `Tags` | A set of strings for grouping and filtering, separated by commas. Optional. |
+| `Auto Approve` | If enabled, all new Service contract requests are automatically approved. Otherwise, Dev Portal admins must manually approve requests. Default: `false`. |
+| `Description` | Description displayed in the information about a Service in the Developer Portal. Optional. |
+| `Display Name` | Unique name displayed in the information about a Service in the Developer Portal. Required. |
+| `Show Issuer` | Displays the Issuer URL in the Service Details page. Default: `false`. **Important:** Exposing the **Issuer URL** is essential for the [Authorization Code Flow](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/3rd-party-oauth/#ac-flow) workflow configured for third-party identity providers. |
 
-Default: `false`
+## Next steps
 
-Displays the Issuer URL in the Service Details. **Note:** Exposing
-the Issuer URL is essential for the
-[Authorization Code Flow](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/3rd-party-oauth/#ac-flow)
-workflow configured for third-party identity providers.
+Kong OAuth2 strategy:
```

Comment:
```suggestion
{{site.base_gateway}} OAuth2 strategy:
```

Reply:
will rework. this rebrand is a bit unwieldy.